components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/util/EventHookHandlerUtils.java

```diff
@@ -338,6 +340,25 @@ public static boolean isParentPolicyImmediateOrgs(String parentTenantDomain) thr
                         PolicyEnum.IMMEDIATE_EXISTING_AND_FUTURE_ORGS.getPolicyCode());
     }
 
+    public static boolean isB2BUserLogin(AuthenticationContext authContext) {
```

Comment:
Add javadoc comments for public methods

Reply:
added https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/81/commits/be18d34d5d2684f08400df2aa28492e38bcb9f3a